packages/gatsby-cli/src/reporter/prepare-stack-trace.ts

```diff
@@ -109,6 +109,19 @@ function getPosition({
   map: BasicSourceMapConsumer | IndexedSourceMapConsumer
   frame: stackTrace.StackFrame
 }): NullableMappedPosition {
+  if (frame.getFileName().includes(`webpack:`)) {
+    // if source-map-register is initiated, stack traces would already be converted
+    return {
+      column: frame.getColumnNumber() - 1,
+      line: frame.getLineNumber(),
+      source: frame
+        .getFileName()
+        .substr(frame.getFileName().indexOf(`webpack:`))
+        .replace(/webpack:\/+/g, `webpack://`),
+      name: null,
+    }
+  }
+
```

Comment:
this is interesting - once we import (directly or indirectly) `gatsby-telemetry` in worker, stack traces are different (due to `@turist/fetch` installing source map handler that map stack traces to source files). This just adjust stack traces produced this way so they are still compatible with our codeframe printer for html generation errors